build/verify-features.ts

```diff
@@ -53,12 +54,31 @@ function findError(filename: string): string | void {
 
 	const featureMeta = featuresInReadme.find(feature => feature.id === featureId);
 	if (!featureMeta) {
-		return `ERR: The feature ${featureId} should be described in the readme`;
+		return `ERR: ${featureId} should be described in the readme`;
 	}
 
 	if (featureMeta.description.length < 20) {
 		return `ERR: ${featureId} should be described better in the readme (at least 20 characters)`;
 	}
+
+	// eslint-disable-next-line @typescript-eslint/prefer-regexp-exec
+	const featureMatch = readmeContent.match(findFeatureRegex(featureId as FeatureID));
+	// eslint-disable-next-line @typescript-eslint/prefer-regexp-exec
```

Comment:
Why?

Reply:
`match` is more concise in this case, as we just need to call it once to get all the matches. `exec` will need us to use a while loop to get all the matches.

Reply:
`exec` seems to be more efficient when I'm counting line numbers too... so I'm using `exec` now.